docs/resources/scenarios/sample-scenario-overview.md

```diff
@@ -0,0 +1,20 @@
+---
+title: 'Sample Scenarios for Office Scripts'
+description: 'An explanation for the larger, scenario-driven samples for Office Scripts in Excel on the web.'
+ms.date: 02/13/2020
+localization_priority: Normal
+---
+
+# Sample Scenarios for Office Scripts
+
+Office Scripts help automate parts of your daily routine. These day-to-day tasks often exist in unique ecosystems, with Excel workbooks that are set up in particular ways. These larger scenario samples demonstrate such real-world use-cases. They include both the Office Scripts and the workbooks, so you can see the scenario from end to end.
+
+## Scenarios
+
+### [Analyze web downloads](analyze-web-downloads.md)
+
+This scenario features a script that parses web traffic records to determine a user's country of origin. It showcases the skills of text parsing, using subfunctions in scripts, applying conditional formatting, and working with tables.
```

Comment:
Since it's uncommon in the docs for section headings to be hyperlinks, users may not realize that the section header is clickable. Perhaps consider removing the hyperlink from the H3, and instead beginning the sentence with:

`The [Analyze web downloads](analyze-web-downloads.md) scenario features a script that..."`